setup.cfg

```diff
@@ -32,7 +32,7 @@ package_dir =
     =src
 packages = find:
 zip_safe = True
-python_requires = >= 3.9
+python_requires = >= 3.7
```

Comment:
Maybe also update the `Programming Language :: Python :: 3.x` classifier?

Reply:
I guess we can, yes!